presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -691,6 +692,12 @@ std::vector<std::string> PrestoServer::registerConnectors(
           fileName.substr(0, fileName.size() - kPropertiesExtension.size());
 
       auto connectorConf = util::readConfig(entry.path());
+      // Generate and store identity for use by filesystem to uniquely identify
+      // a catalog even if the connector.name is same.
+      connectorConf.emplace(
+          kFileSystemIdentity,
+          fmt::format("{}-{}", fileName, connectorConf[kConnectorName]));
```

Comment:
@vermapratyush 
I think we should use "catalogName" here, not "fileName"?
We don't need the extension.

Reply:
Updated.